src/doc/unstable-book/src/language-features/intra-doc-pointers.md

```diff
@@ -0,0 +1,15 @@
+# `intra-doc-pointers`
+
+The tracking issue for this feature is: [#80896]
+
+[#80896]: https://github.com/rust-lang/rust/issues/80896
+
+------------------------
+
+Rustdoc does not currently allow disambiguating `*const` and `*mut`, and using
+pointers/references in intra-doc links are unstable until it does.
+
+```rust
+#![feature(intra_doc_pointers)]
+//! [pointer::add]
```

Comment:
Do you want to use something like

```suggestion
//! [`pointer::add`]
```

or

```suggestion
//! [`*const::add`]
```

instead? Not a big deal, but might be nicer.

Reply:
I don't think so, no. These doc-comments will never be rendered in HTML and I want to avoid giving the impression that backticks are related to the feature somehow.